docs/tutorials/customize_runtime.md

```diff
@@ -108,7 +108,7 @@ class MyOptimizerConstructor(object):
 
 ```
 
-The default optimizer constructor is implemented [here](https://github.com/open-mmlab/mmcv/blob/9ecd6b0d5ff9d2172c49a182eaa669e9f27bb8e7/mmcv/runner/optimizer/default_constructor.py#L11), which could also serve as a template for new optimizer constructor.
```

Comment:
The version of mmcv is out-of-date. So I updated it to the newest version (with a fixed version instead of the master branch to avoid the line linking confusion).